### ISSUE TITLE
I cannot choose [--best_checkpoint_metric BLEU]

### ISSUE THREAD

**Issue by moritay-gm** [pytorch/fairseq#1449]
In translation task, I have to choice "best_checkpoint_metric".
But, my goal is to get the best BLEU score model in validation dataset.
So, I want to choice BLEU (not "loss") as "best _checkpoint_metric", because the lowest "loss" score model in validation dataset does not necessarily perform the best from the viewpoint of BLEU.

But  I cannot choose BLEU as best_checkpoint_metric.

Or, is it natural to use the lowest loss model's BLEU score as metric in NLP conference's paper??

**Comment by lematt1991** [comment]
Currently there is no way to use BLEU as your validation metric.  It is common to use lowest validation loss for choosing checkpoints.
